cedar-wasm/build-wasm.sh

```diff
@@ -84,6 +84,9 @@ process_types_file() {
     s/{[[:space:]]*\.: /{ ".": /g
     s/ | __skip//g
     s/ { .*: __skip } |//g
+    # Workaround for wasm-bindgen issue resulting in malformed return type Array
+    # without generic paramter. See https://github.com/rustwasm/wasm-bindgen/issues/4207
+    s/[[:space:]]Array;/ number[];/g
```

Comment:
Do we need to patch just once? If so, maybe we should make sure this is being replaced only once or have a more focused way to apply the patch.